### PR TITLE
added unit nitpick

### PR DESCRIPTION
**Pull request by nabobalis**
Also fix for the drms 0.8.0 release which changes the API docs

### CODE REVIEW

File: sunpy/image/tests/test_transform.py [hunk 1]
```diff
@@ -245,6 +245,7 @@ def test_float32(identity):
     assert np.issubdtype(out_arr.dtype, np.float32)
 
 
+@skip_windows
```
Comment (nabobalis) [inline sunpy/image/tests/test_transform.py:248]: This is now broken on windows and I don't have the skills to debug why.

Comment (nabobalis) [reply sunpy/image/tests/test_transform.py:248]: I should open an issue to track this.

Comment (wtbarnes) [reply sunpy/image/tests/test_transform.py:248]: This makes me nervous but ok

**Review by wtbarnes — APPROVED** [review]

**Review by alasdairwilson — APPROVED** [review]

File: sunpy/image/tests/test_transform.py (hunk 1 above)
Comment (nabobalis) [reply sunpy/image/tests/test_transform.py:248]: We will have to track this down but its a numpy issue on windows. Nothing we can do to fix that outside of reporting it upstream.

**Pull request merged**